Source/Magick.NET/Shared/Profiles/Exif/ExifDefinition.cs

```diff
@@ -0,0 +1,364 @@
+            { ExifTag.Indexed, new ExifDefinition(ExifTag.Indexed, ExifDataType.Short, false) },
+            { ExifTag.JPEGTables, new ExifDefinition(ExifTag.JPEGTables, ExifDataType.Undefined, false) },
+            { ExifTag.OPIProxy, new ExifDefinition(ExifTag.OPIProxy, ExifDataType.Short, false) },
+            { ExifTag.JPEGProc, new ExifDefinition(ExifTag.JPEGProc, ExifDataType.Long, false) },
+            { ExifTag.JPEGInterchangeFormat, new ExifDefinition(ExifTag.JPEGInterchangeFormat, ExifDataType.Long, false) },
+            { ExifTag.JPEGInterchangeFormatLength, new ExifDefinition(ExifTag.JPEGInterchangeFormatLength, ExifDataType.Long, false) },
+            { ExifTag.JPEGRestartInterval, new ExifDefinition(ExifTag.JPEGRestartInterval, ExifDataType.Short, false) },
+            { ExifTag.JPEGLosslessPredictors, new ExifDefinition(ExifTag.JPEGLosslessPredictors, ExifDataType.Short, false) },
+            { ExifTag.JPEGPointTransforms, new ExifDefinition(ExifTag.JPEGPointTransforms, ExifDataType.Short, false) },
+            { ExifTag.JPEGQTables, new ExifDefinition(ExifTag.JPEGQTables, ExifDataType.Long, false) },
+            { ExifTag.JPEGDCTables, new ExifDefinition(ExifTag.JPEGDCTables, ExifDataType.Long, false) },
+            { ExifTag.JPEGACTables, new ExifDefinition(ExifTag.JPEGACTables, ExifDataType.Long, false) },
+            { ExifTag.YCbCrCoefficients, new ExifDefinition(ExifTag.YCbCrCoefficients, ExifDataType.Rational, false) },
+            { ExifTag.YCbCrSubsampling, new ExifDefinition(ExifTag.YCbCrSubsampling, ExifDataType.Short, false) },
+            { ExifTag.YCbCrPositioning, new ExifDefinition(ExifTag.YCbCrPositioning, ExifDataType.Short, false) },
+            { ExifTag.ReferenceBlackWhite, new ExifDefinition(ExifTag.ReferenceBlackWhite, ExifDataType.Rational, false) },
+            //{ ExifTag.XMLPacket, new ExifDefinition(ExifTag.XMLPacket, ExifDataType.Byte, false) },
+            { ExifTag.Rating, new ExifDefinition(ExifTag.Rating, ExifDataType.Short, false) },
+            { ExifTag.RatingPercent, new ExifDefinition(ExifTag.RatingPercent, ExifDataType.Short, false) },
+            { ExifTag.ImageID, new ExifDefinition(ExifTag.ImageID, ExifDataType.Ascii, false) },
+            { ExifTag.CFARepeatPatternDim, new ExifDefinition(ExifTag.CFARepeatPatternDim, ExifDataType.Short, false) },
+            { ExifTag.CFAPattern, new ExifDefinition(ExifTag.CFAPattern, ExifDataType.Byte, false) },
+            { ExifTag.BatteryLevel, new ExifDefinition(ExifTag.BatteryLevel, ExifDataType.Rational, false) },
+            { ExifTag.Copyright, new ExifDefinition(ExifTag.Copyright, ExifDataType.Ascii, false) },
+            { ExifTag.ExposureTime, new ExifDefinition(ExifTag.ExposureTime, ExifDataType.Rational, false) },
+            { ExifTag.FNumber, new ExifDefinition(ExifTag.FNumber, ExifDataType.Rational, false) },
+            //{ ExifTag.IPTCNAA, new ExifDefinition(ExifTag.IPTCNAA, ExifDataType.Long, false) },
```

Comment:
Can you clarify why this is commented out?

Reply:
I took a list of Exif values from a version of the spec and used a macro to convert them to the code here. The entries which are commented out are in the spec, but are not in the ExifTag enumeration. I've moved them in the next push so that they are grouped at the end of the class as an aide memoire. I'll stick a //TODO on it, too.